app/controlplane/api/workflowcontract/v1/crafting_schema_validations.go

```diff
@@ -30,6 +30,7 @@ import (
 // detection. The order of the list is important as it defines the order of the
 // detection process. Normally from most common one to the least common one and weaker validation method.
 var CraftingMaterialInValidationOrder = []CraftingSchema_Material_MaterialType{
+	CraftingSchema_Material_ZAP_DAST_ZIP,
```

Comment:
why first? Isn't it too expensive? I'd make sure we order by less expensive to more

Reply:
Yes, I don't know why I placed it there, it should be lower on the list.